src/views/forms/AccountManagementForms/WithdrawForm.tsx

```diff
@@ -396,6 +385,17 @@ export const WithdrawForm = () => {
         }),
       };
 
+    if (routeErrors) {
```

Comment:
route errors shouldn't show up before local validation errors like not having a wallet address